probe/base/metrics.go

```diff
@@ -25,18 +25,21 @@ import (
 
 // metrics is the probe metrics
 type metrics struct {
-	Total    *prometheus.CounterVec
-	Duration *prometheus.GaugeVec
-	Status   *prometheus.GaugeVec
-	SLA      *prometheus.GaugeVec
+	TotalCnt  *prometheus.GaugeVec
```

Comment:
It is suggested not to change the name of this metric due to compatibility issues. In considering compatibility issues, the names of metrics that are already in use should not be changed. If the name of a metric is changed, it may cause problems in existing code, documentation, tools, or systems. This can result in errors, confusion, or other issues, so the impact should be carefully considered before making any changes to the name